extensions/positron-python/python_files/positron/positron_ipykernel/positron_ipkernel.py

```diff
@@ -438,6 +438,27 @@ def __init__(self, **kwargs) -> None:
             module="jedi",
         )
 
+        # TODO: We could move this to its own function in a separate module.
```

Comment:
Nit: Do you mind pulling this out into a function, something like `set_holoviews_extension` in a new `holoviews.py` module? That better fits the current module structure.